pkg/component/clusterautoscaler/bootstrap.go

```diff
@@ -63,9 +64,23 @@ func (b *bootstrapper) Deploy(ctx context.Context) error {
 				},
 			},
 		}
+
+		role = &rbacv1.Role{
```

Comment:
I don't understand this `Role`. It would allow CA to read deployments in the `garden` namespace, but there is no CA running in this namespace.